redisinsight/ui/src/pages/vector-search/create-index/VectorSearchCreateIndex.tsx

```
@@ -34,24 +35,45 @@ import {
 import successMessages from 'uiSrc/components/notifications/success-messages'
 import { parseCustomError } from 'uiSrc/utils'
 import { Row } from 'uiSrc/components/base/layout/flex'
+import { Banner } from 'uiSrc/components/base/display'
 
-const stepNextButtonTexts = [
-  'Proceed to adding data',
-  'Proceed to index',
-  'Create index',
+const stepNextButton: IStepNextButton[] = [
+  {
+    text: 'Proceed to adding data',
+    testId: 'proceed-to-adding-data-button',
+  },
+  {
+    text: 'Proceed to index',
+    testId: 'proceed-to-index-button',
+  },
+  {
+    text: 'Create index',
+    testId: 'create-index-button',
+  },
 ]
 
 export type VectorSearchCreateIndexProps = {
   initialStep?: number
 }
 
+const DisabledBannerText = () => (
+  <Text component="div">
```

Note: Maybe we can add a test to verify the presence of this banner. Now it may seem meaningless, since this is the only flow and it's always visible, but in the next iterations, we’re going to add different types of data (not only preset) and we might show this conditionally, so maybe it won't harm us if we have a test to verify that.

---

Added the test. Since we don't have a form to choose preset / custom data and sampleDataType has a hardcoded value, I only considered the final step for the test. I suppose when we introduce this functionality, this test will fail and we will be forced to enhance it with the sampleDataType setup.